notebook/utils.py

```diff
@@ -24,6 +24,13 @@
 # It is used by BSD to indicate hidden files.
 UF_HIDDEN = getattr(stat, 'UF_HIDDEN', 32768)
 
+# Replace `os.stat` which can't stat a host mapped volume from inside a Windows Container.
+WINDOWS_CONTAINER = sys.platform == 'win32' and os.environ.get('USERNAME') == 'ContainerAdministrator'
+if WINDOWS_CONTAINER:
+    f_stat = os.lstat
```

Comment:
Can we do this for Windows in general, and remove the WINDOWS_CONTAINER constant?